tests/lib/config/config-ops.js

```diff
@@ -481,4 +481,151 @@ describe("ConfigOps", function() {
 
     });
 
+    describe("normalize()", function() {
+        it("should convert error rule setting to 2 when rule has just a severity", function() {
+            var config = {
+                rules: {
+                    foo: "error",
+                    bar: "error"
+                }
+            };
+
+            ConfigOps.normalize(config);
+
+            assert.deepEqual(config, {
+                rules: {
+                    foo: 2,
+                    bar: 2
+                }
+            });
+        });
+
+        it("should convert error rule setting to 2 when rule has array with severity", function() {
+            var config = {
+                rules: {
+                    foo: ["error", "something"],
+                    bar: "error"
```

Comment:
I have two rules because I'm always afraid of side effects that happens with multiple input types.

There are only two things to test: rules with arrays and rules without arrays. 

Remember, all this method does is translate string values into numeric values, it's not very complicated.